privval/remote_signer.go

```diff
@@ -229,7 +229,10 @@ func handleRequest(req RemoteSignerMsg, chainID string, privVal types.PrivValida
 	switch r := req.(type) {
 	case *PubKeyMsg:
 		var p crypto.PubKey
-		p = privVal.GetPubKey()
+		p, err = privVal.GetPubKey()
+		if err != nil {
+
```

Comment:
??? can we add at least a comment

Reply:
Sorry, I missed that one. I'll return the error. In a follow-ip PR we should split up `PubKeyMsg` into `PubKeyRequest` and `PubKeyResponse`. The latter should contain the error as with the other requests below. 
Thanks a lot for the through review!